metadata/src/test/java/edu/unc/lib/dl/util/DepositStatusFactoryTest.java

```diff
@@ -0,0 +1,38 @@
+package edu.unc.lib.dl.util;
```

Comment:
Could you add the license header to this file?  You can find it in most of the other cdr classes